spec/controllers/account/companies_controller_spec.rb

```diff
@@ -39,20 +39,28 @@
       expect(response).to render_template(:show)
     end
 
+    it "should populate a twitter" do
```

Comment:
it assigns twitter to a twitter processor

Reply:
here it's trying to connect to external service. I think you don't have any stub here

Reply:
doesn't it create an object only, without starting the methods to external services?